content/docs/dvclive/live/log_image.md

```diff
@@ -39,14 +39,18 @@ Supported values for `val` are:
   should be in a format that is readable by
   [`PIL.Image.open()`](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.open)
 
-The images will be saved in `{Live.plots_dir}/images/{name}`:
+The images will be saved in `{Live.plots_dir}/images/{name}`. When using
+`Live(cache_images=True)`, the images will also be <abbr>cached</abbr> as part
+of `Live.end()` and a `.dvc` file will be saved to track them:
 
 ```
 dvclive
 └── plots
-    └── images
-        ├── numpy.png
-        └── pil.png
+    ├── images
+    │   ├── numpy.png
+    │   ├── pil.png
+    │   └── sample.png
+    └── images.dvc
```

Comment:
Minor: maybe add / mention gitignore here?

Feels like implementation detail but somehow relevant to understand the behavior, specially for newcomers not familiar with DVC data management

Reply:
Ah, good idea. It's not shown by `tree` by default so I forgot it.